assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/styles.scss

```diff
@@ -8,6 +8,41 @@
 			padding: 1rem;
 			height: 100px;
 		}
+		tr {
+			background: var(--ee-color-grey-14);
+			&:hover {
+				background: var(--ee-color-grey-13);
+			}
+			&:nth-child(even) {
+				background: var(--ee-color-grey-13);
+				&:hover {
+					background: var(--ee-color-grey-12);
+				}
+			}
+		}
+		td {
+			&.date-cell {
+				border-right: 5px solid var(--ee-color-white);
+				padding: var(--ee-padding-small);
+				width: 300px;
+				position: relative;
+				.date-stripe {
+					display: inline-block;
+					height: 100%;
+					width: 5px;
+					position: absolute;
+					top: 0;
+					left: -5px;
+				}
+				.date-cell__name {
```

Comment:
why not:

```suggestion
				&__name {
```

or does that target things differently?

Reply:
yes, that was targeting the `td`, like this:

![image](https://user-images.githubusercontent.com/1477453/78672127-e2bcef00-78e8-11ea-8e34-e55888dcd05c.png)

Reply:
ended up adding `date-cell-content` classname to be able to target `date-cell-content__name`, hope that's fine

Reply:
totally fine, thnx for the explanation 👍